src/signup/signup.hellpers.js

```diff
@@ -0,0 +1,24 @@
+const fsPromises = require('fs').promises;
+const { signUpDb } = require('../../config');
+
+
+const writeUser = async (user, res) => {
+
+  const userWithId = {
+    _id: Date.now(),
+    ...user
+  }
+
+  try {
+    await fsPromises.writeFile(`${signUpDb}${user.username}.json`, JSON.stringify(userWithId));
```

Comment:
better to concatenate path's with `path.join`

Reply:
add path.join